code/datums/emergency_calls/beacon-reinforcements/rmc.dm

```diff
@@ -0,0 +1,57 @@
+/datum/emergency_call/royal_marines/beacon
+	name = "Royal Marines Commandos (Beacon)"
+	mob_max = 8
+	mob_min = 1
+	spawn_max_amount = TRUE
+	home_base = /datum/lazy_template/ert/twe_station
+	shuttle_id = MOBILE_SHUTTLE_ID_ERT4
+	name_of_spawn = /obj/effect/landmark/ert_spawns/distress_twe/beacon
+	item_spawn = /obj/effect/landmark/ert_spawns/distress_twe/item
+
+/datum/emergency_call/royal_marines/beacon/New()
+	..()
+	arrival_message = "[MAIN_SHIP_NAME], this is the HMS Kurtz of the Three World Empires. We've received your call and are enroute to aid."
```

Comment:
Kurtz is a CBRN ship. Thunderchild is the HMS ship used in cm. You cal call it whatever though.
```suggestion
	arrival_message = "[MAIN_SHIP_NAME], this is the HMS Thunderchild of the Three World Empires. We've received your call and are enroute to aid."
```

Reply:
changed it up to the HMS Thanes. I want to keep it as a different name than the other two ERT ships so I don't have to add anything to check if a ship name has been taken

Reply:
The USS Kurtz map can be used for any USCM faction ERT, not just CBRN.